### ISSUE TITLE
Add 'accessibility options' to the Bootstrap template

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1389]
It would be good to have something similar to the 'accessibility options' dialog in the Bootstrap project that gave end-users the option to change the default theme that the project uses. We would also need new themes developing: dark mode, light mode, high contrast & black on yellow to be consistent with the XOT project options.
